test/HandleHumanLocalMUnitTests.hs

```diff
@@ -0,0 +1,86 @@
+module HandleHumanLocalMUnitTests (handleHumanLocalMUnitTests) where
+-- at some point I'm guessing we'll want our unit test hierarchy to match the file hierarchy
+
+import Prelude ()
+
+import Game.LambdaHack.Core.Prelude
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+
+import           Game.LambdaHack.Client.UI.HandleHelperM
+import           Game.LambdaHack.Client.UI.HandleHumanLocalM
+import qualified Game.LambdaHack.Client.UI.HumanCmd as HumanCmd
+
+import           Game.LambdaHack.Common.ActorState
+import           Game.LambdaHack.Common.Item
+import           Game.LambdaHack.Common.ItemAspect
+import           Game.LambdaHack.Common.Level
+import           Game.LambdaHack.Common.PointArray as PointArray
+import           Game.LambdaHack.Common.Point
+import           Game.LambdaHack.Common.ReqFailure
+import           Game.LambdaHack.Content.TileKind
+import           Game.LambdaHack.Definition.DefsInternal ( toContentId, toContentSymbol )
+import           Game.LambdaHack.Definition.Flavour
+
+import           UnitTestHelpers
+
+
+stubItem :: Item
+stubItem = Item { jkind = IdentityObvious (toContentId 0), jfid = Nothing, jflavour = dummyFlavour }
+
+testItemFull :: ItemFull
+testItemFull = ItemFull { itemBase = stubItem, itemKindId = toContentId 0, itemKind = testItemKind, itemDisco = ItemDiscoFull emptyAspectRecord, itemSuspect = False }
+
+
+handleHumanLocalMUnitTests :: TestTree 
+handleHumanLocalMUnitTests = testGroup "handleHumanLocalMUnitTests" 
+  [ testCase "verify stubLevel has tile element" $
+    do let level = stubLevel -- (Just level) = EM.lookup (toEnum 0) (sdungeon testState)
```

Comment:
I'd do `case EM.lookup (toEnum 0) (sdungeon testState) of` and fail the test when I get `Nothing` and continue with `Just level`.